packages/transformers/js/core/src/env_replacer.rs

```diff
@@ -41,6 +41,38 @@ impl<'a> Fold for EnvReplacer<'a> {
       }
     }
 
+    if let Expr::Bin(ref binary) = node {
+      if let Expr::Member(ref member) = &*binary.right {
+        if let Expr::Lit(ref lit) = &*binary.left {
+          if let Lit::Str(ref left) = lit {
```

Comment:
I'm getting some clippy warnings that 
1. the match can be replaced with an `if let` because there's only one branch.
2. you can save one `if let` by doing `if let Expr::Lit(Lit::Str(ref left))`.

Reply:
reasonable!! Thank you! 👍  [c45bcb7](https://github.com/parcel-bundler/parcel/pull/7954/commits/c45bcb791b3c8afa4d39df1068d9a7e87fe93d33)